java/ql/lib/qlpack.yml

```diff
@@ -7,6 +7,8 @@ library: true
 upgrades: upgrades
 dependencies:
   codeql/regex: ${workspace}
+  codeql/typetracking: ${workspace}
+  codeql/util: ${workspace}
```

Comment:
If this is only a transitive dependency, I would hope it doesn't have to be added explicitly.

Reply:
Right. It was a direct dependency at some point during development, but ended up being just transitive. It'll soon enough become a direct dependency, so I think we might as well leave it here.